notebooks/Understand_Tables.ipynb

```diff
@@ -0,0 +1,4387 @@
+{
```

Comment:
<ul><li>"Here's that table looks like close up: " - "Here's what that table looks like close up:"</li><li>Just wondering if we should include a (short) sentence describing what Watson Discovery is (and perhaps hyperlink also)?</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/107/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Good idea! I've tweaked the intro a bit to include a more direct reference to Watson Discovery.</p>